addons/bluguide/masterlist.lua

```diff
@@ -0,0 +1,199 @@
+--Copyright � 2015, Anissa
+--All rights reserved.
+
+--Redistribution and use in source and binary forms, with or without
+--modification, are permitted provided that the following conditions are met:
+
+--    * Redistributions of source code must retain the above copyright
+--      notice, this list of conditions and the following disclaimer.
+--    * Redistributions in binary form must reproduce the above copyright
+--      notice, this list of conditions and the following disclaimer in the
+--      documentation and/or other materials provided with the distribution.
+--    * Neither the name of bluGuide nor the
+--      names of its contributors may be used to endorse or promote products
+--      derived from this software without specific prior written permission.
+
+--THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+--ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+--WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+--DISCLAIMED. IN NO EVENT SHALL ANISSA BE LIABLE FOR ANY
+--DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+--(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+--LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+--ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+--(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+--SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+texts = require("texts")
+spellbuttons = require("ui/spellbuttons")
+spelllist = {}
+display = {}
+
+local function get_limits()
+    local player = windower.ffxi.get_player()
+    local slots = 0
+    local points = 0
+    local gifts = 0
+    local level = 0
+    
+    if player.main_job == "BLU" then
+        if player.main_job_level > 70 then
+            slots = 20
+        else
+            slots = (math.floor((player.sub_job_level + 9) / 10) * 2) + 4
+        end
+        
+        points = (math.floor((player.main_job_level + 9) / 10) * 5) + 5
+        if player.main_job_level >= 75 then
+            points = points + player.merits.assimilation
+            if player.main_job_level == 99 then
+                points = points + player.job_points.blu.blue_magic_point_bonus
+            end
+        end
+        level = player.main_job_level
+    elseif player.sub_job == "BLU" then
+        points = (math.floor((player.sub_job_level + 9) / 10) * 5) + 5
+        slots = (math.floor((player.sub_job_level + 9) / 10) * 2) + 4
+        level = player.sub_job_level
+    else
+        magicpoints = 0
+        spelllevel = 0
+        spellslots = 0
```

Comment:
Here these 3 are called magicpoints spelllevel  and spellslots, but in the other 2  parts of the if statement they're just points, level and slots. Just wanna make sure this is intentional.